### DISCUSSION TITLE
How do I set a different theme color for:

### DISCUSSION THREAD

**Discussion by Crehet** [martinrotter/rssguard#1365] [Q&A] [answered]
I grabbed the color and tried to search for all files for any contents that could contain `#DBDAE2` or `rgba(219, 218, 226, 1)` but I get no results. I've got no idea how to theme that specific widget.

![file](https://github.com/martinrotter/rssguard/assets/119722248/00d7ccc8-20d5-4183-8e4f-02f7be26ab3b)

**Comment by martinrotter** [comment, marked as answer]
https://rssguard.readthedocs.io/en/stable/features/skins.html

To theme that particular border for exactly only and only these two widgets, you have to edit skin file `qt_style.qss` and use `TYPE-NAME#OBJECT-NAME` CSS selector syntax.

Like this:

```css
QListWidget#m_listSettings, QStackedWidget#m_stackedSettings {
    border: 1px solid red;
}
```

So those two objects are called "m_listSettings" and "m_stackedSettings".

Pics:

![image](https://github.com/martinrotter/rssguard/assets/1255302/59b5e547-99a4-42ca-9f95-846702f0cab9)
![image](https://github.com/martinrotter/rssguard/assets/1255302/62ef43f9-c24c-42ac-89e1-f7e670aa5759)
![image](https://github.com/martinrotter/rssguard/assets/1255302/84b5b911-060d-4c7d-9115-61023456ef9e)
![image](https://github.com/martinrotter/rssguard/assets/1255302/095314c2-0d43-4117-a484-20352f0f6ef1)

**Comment by Crehet** [comment]
what tool did you use to inspect the widgets? i tried googling for a qt widget inspector, like how web browser do. and installed gammaray to try but the thing was not working out of the box and left it there. 

im using rssguar-lite, would the webengine version allow me to see those properties, like `m_listSettings` and `m_stackedSettings`?

also, i figured out that uncheking `Use skin colors` made rssguard feel more proper, like as if it were now more loyal to my KDE color theme. 
![file](https://github.com/martinrotter/rssguard/assets/119722248/0ee0f808-1d5c-43c0-a676-257007f39f0e)
